examples/cylinder/3d_unsteady_discrete/optimize/README.md

```diff
@@ -0,0 +1,24 @@
+[//]: <> (title: Flow around a cylinder use case tutorial, author: Xiandong Liu @liuxiandong at baidu.com)
+
+
+# Flow around a Cylinder
+
+This guide introduces to how to build a PINN model for simulating the flow around a cylinder in PaddleScience.
+
+This demo is based on Paddle static graph mode, including the following parts:
+
+ - Use v1.0 API, including geometry, pde, bc, network, algorithm and other parts;
```

Comment:
Use the 1.0 API of PaddleScience suite to complete the construction of geometry, PDE, BC, network, algorithm, etc

Reply:
Done.